Common/Gfx/Shape.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
Can you remove this? `#pragma once` and the `#ifndef RM_GFX_SHAPE_H_` check below are equivalent. We use the latter.